presto-hive/src/main/java/com/facebook/presto/hive/HiveClientModule.java

```diff
@@ -299,11 +301,19 @@ public StripeMetadataSourceFactory createStripeMetadataSourceFactory(OrcCacheCon
                     .expireAfterAccess(orcCacheConfig.getStripeStreamCacheTtlSinceLastAccess().toMillis(), MILLISECONDS)
                     .recordStats()
                     .build();
+            Cache<StripeStreamId, List<RowGroupIndex>> rowGroupIndexCache = CacheBuilder.newBuilder()
```

Comment:
orc.stripe-metadata-cache-enabled is enabled by default in some raptorX clusters. But we may not want to enable the new RowGroupIndex by default without testing it properly. Maybe add a new session property to control it?